### ISSUE TITLE
hey, i been following this book to but 

### ISSUE THREAD

**Issue by cyrstem** [zackaLee/videoSynth#1]
i have an issue with it  when  i try to load the xml file the gui doesn't work anymore i try with the example files and doesn't work either
can you help me?
fix this

**Comment by zackaLee** [comment]
I can try to help for sure. What happens with the xml file?

**Comment by cyrstem** [comment]
Hey thanks im not sure the file is save but never loaded i try with your example and  is the same thing  the gui won't work anymore every time i try to upload the xml and i get this message in the console 
[ error ] ofXml: loadFromBuffer(): DOM ERROR   Ans not sure whta could it be or why this happen

**Comment by zackaLee** [comment]
I know that I had some trouble with the settings.xml and loading and saving
for a while.

I think the first thing that was giving me troubles was having
read,write,execute permissions on the project folder. I'm on Linux and I
gave the project folder a chmod -R 0777 'folder-name'.  If iI remember,
this is what fixed the loading and saving troubles.

Let me know how you go,
Cheers

**Comment by cyrstem** [comment]
yeah this is the part that doesn't work

here is my code in the key events functions.
// --------------------------
if ( key == 's' ) {
ofFileDialogResult res;
res = ofSystemSaveDialog( "preset.xml", "Saving Preset");
if ( res.bSuccess ) gui.saveToFile( res.filePath );
}
if ( key == 'l' ) {
ofFileDialogResult res;
res = ofSystemLoadDialog( "Loading Preset" );
if ( res.bSuccess ) gui.loadFromFile( res.filePath );
}
try changing the chmod didn't work either jaja i have no idea what im doing wrong

**Comment by zackaLee** [comment]
hey,
does your project folder contain 'bin/data'?

**Comment by zackaLee** [comment]
Also, can you let me know what operating system you are using as well as
version of openFrameworks?

**Comment by cyrstem** [comment]
Yeah it does have a bin/data and the os is yosemite 10.10.5 mac os 
My OF vs is 8.4 
I try with the 9 same thing

**Comment by cyrstem** [comment]
Im begining to think is my xcode or something

**Comment by zackaLee** [comment]
Im not sure about the mac setup mate.

Your code is correct for loading and saving xmls.

Other than that, maybe check how the permissions are handled with xcode.

From memory, if your 'bin/data' does not have a settings.xml in it then for
me, it was because i did not have permission to write in the directory.

Keep in touch as to how you go. Good luck

Zack

**Comment by cyrstem** [comment]
well i manage to fix it kind of,  i try in the new openframework 0.9 and it work so i guess it has something to do with the 8.4 on mac  ill try to figure out  what was on the 8.4

**Comment by zackaLee** [comment]
Ah cool,

I'm happy it works for you. How are you finding the book?

It was really fun going through all the examples in that for me :) I really
learn the best making cool stuff and it was perfect for that, even though
the majority of stuff went over my head.

Have fun with it

**Comment by cyrstem** [comment]
yeah i think that's the coolest thing of this book, how they just show you how to  do it step by step right now i'm on the 3d staff  it's really great
